example/dummy/persistent_dummy.go

```diff
@@ -38,14 +42,19 @@ func NewPersistentDummyApplication(dbDir string) *PersistentDummyApplication {
 	stateTree := iavl.NewIAVLTree(0, db)
 	stateTree.Load(lastBlock.AppHash)
 
-	log.Notice("Loaded state", "block", lastBlock.Height, "root", stateTree.Hash())
+	// log.Notice("Loaded state", "block", lastBlock.Height, "root", stateTree.Hash())
```

Comment:
Fair point. It's not a critical log message to have. I've also considered moving the tree loading outside this contructor, so it would take the tree as an arg. But then we still have the ugly SetLogger issue (which isnt really a big deal ...)